hera_cal/utils.py

```diff
@@ -1318,7 +1318,10 @@ def red_average(data, reds=None, bl_tol=1.0, inplace=False,
             # take the weighted average
             wsum = np.sum(w, axis=0).clip(1e-10, np.inf)  # this is the normalization
             davg = np.sum(d * w, axis=0) / wsum  # weighted average
-            fmax = np.max(f, axis=2)             # collapse along freq: marks any fully flagged integrations
+            # set flagged data to 1.0+0.0j
```

Comment:
@jsdillon @aewallwi we are proposing to change the behavior of `red_average` such that pixels that are fully flagged are set to 1.0 instead of 0.0. this is more consistent with what the LST binner does. can you comment on any possible ramifications of this?

In theory, any downstream block that honors flags should not be affected. the reason to move to 1.0 instead of 0.0 is that it is 1. consistent with lst binner and 2. plays nicer with division

Reply:
I think this should be fine. I approve.